Public/Sdk/Public/Prelude/Prelude.Configuration.Resolvers.dsc

```diff
@@ -142,7 +142,7 @@ interface MsBuildResolver extends ResolverBase, UntrackingSettings {
      * Optional file name for the project or solution that should be used to start parsing (under the root traversal)
      * If not provided, BuildXL will try to find a candidate under the root traversal
      */
-    fileNameEntryPoint?: PathAtom;
+    fileNameEntryPoints?: RelativePath[];

```

Comment:
>RelativePath [](start = 26, length = 12)

And please add in the comment relative to what these paths are going to be interpreted